<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Table</class>
 <widget class="QWidget" name="Table">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>餐桌界面</string>
  </property>
  <widget class="QPushButton" name="btn_1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 40px;  border: 2px groove gray;
background-color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>1号桌</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 40px;  border: 2px groove gray;
background-color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>2号桌</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_5">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>30</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 40px;  border: 2px groove gray;
background-color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>5号桌</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_6">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>30</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 40px;  border: 2px groove gray;
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>6号桌</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 40px;  border: 2px groove gray;
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>7号桌</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_8">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 40px;  border: 2px groove gray;
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>8号桌</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>1-6号桌为2人座</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_9">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>180</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 40px;  border: 2px groove gray;background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>9号桌</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_10">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>180</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 40px;  border: 2px groove gray;
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>10号桌</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_11">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>180</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 40px;  border: 2px groove gray;
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>11号桌</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_12">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>180</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 40px;  border: 2px groove gray;background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>12号桌</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_13">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 40px;  border: 2px groove gray;
background-color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>13号桌</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>260</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>7-12号桌为4人座</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_14">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>330</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 40px;  border: 2px groove gray;
background-color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>14号桌</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_15">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>330</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 40px;  border: 2px groove gray;
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>15号桌</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_16">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>330</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 40px;  border: 2px groove gray;
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>16号桌</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_20">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>480</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 40px;  border: 2px groove gray;
</string>
   </property>
   <property name="text">
    <string>20号桌</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_18">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>330</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 40px;  border: 2px groove gray;
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>18号桌</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>410</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>13-18号桌为6人座</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_19">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>480</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 40px;  border: 2px groove gray;
</string>
   </property>
   <property name="text">
    <string>19号桌</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_22">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>480</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 40px;  border: 2px groove gray;
</string>
   </property>
   <property name="text">
    <string>22号桌</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_23">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>480</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 40px;  border: 2px groove gray;
</string>
   </property>
   <property name="text">
    <string>23号桌</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_21">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>480</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 40px;  border: 2px groove gray;
</string>
   </property>
   <property name="text">
    <string>21号桌</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_24">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>480</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 40px;  border: 2px groove gray;
</string>
   </property>
   <property name="text">
    <string>24号桌</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_17">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>330</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 40px;  border: 2px groove gray;
background-color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>17号桌</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 40px;  border: 2px groove gray;
background-color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>3号桌</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>30</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 40px;  border: 2px groove gray;
background-color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>4号桌</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>560</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>19-24号桌为8人座</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
